src/components/common/NavComponent.js

```diff
@@ -1 +1,48 @@
 // Navigation Component
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+
+class Navbar extends Component {    
+
+    constructor(props) {
+        super(props);
+        this.state = {isActive: false};
+        this.toggleMenu = this.toggleMenu.bind(this);
+    }
+
+    toggleMenu = () => {
+        this.setState({isActive: !this.state.isActive});
+    };
+
+    render() {
+        return (
+            <div>
+                <nav className="navbar">
+                    <div className="navbar-brand">
+                        <h1>&lt; Jrs /&gt;</h1>
+
+                        <div className="navbar-burger"  onClick={this.toggleMenu}>
+                            <span></span>
+                            <span></span>
+                            <span></span>
+                        </div>
+                    </div>
+
+                    <div className={this.state.isActive ? "navbar-menu is-active" : "navbar-menu"}>
```

Comment:
I think the `is-active` logic is intended for the navbar-burger for the x looking thing

Reply:
ah, right, they add is-active to both menu and burger